Library/Homebrew/dev-cmd/update-license-data.rb

```diff
@@ -35,6 +35,6 @@ def update_license_data
     ohai "git add"
     safe_system "git", "add", SPDX::JSON_PATH
```

Comment:
```suggestion
    safe_system "git", "add", SPDX::JSON_PATH/"spdx_licenses.json", SPDX::JSON_PATH/"spdx_exceptions.json"
```

Might be a good idea to rename `JSON_PATH` to something else

Reply:
How about `JSON_DIR` or `DATA_DIR`?

I'd like this to be a path for the directory where the `spdx_licenses.json` and `spdx_exceptions.json` files go. I'd be open to adding constants for the specific files as well but I'm not sure it's necessary.

Reply:
> How about `JSON_DIR` or `DATA_DIR`?

`DATA_DIR` sounds good, although at that point it's not really something specific to SPDX only, so maybe move the constant somewhere else

> I'd be open to adding constants for the specific files as well but I'm not sure it's necessary.

I feel like we should use constants instead of using the raw literals `"spdx_licenses.json"` and `"spdx_exceptions.json"` in various different places